documentation/how-to/navigation.en.md

```diff
@@ -44,3 +47,10 @@ To activate this auto tracking feature, you can simply tap on the positioning bu
 !![](../assets/images/navigation-auto-tracking.png)
 
 This can be described as a simple *staking mode* functionality.
+
+## Positioning precise view
```

Comment:
Should we add "stakeout" to the title? This is what people usually look for

Reply:
+1 to add stakeout as a keyword in there, that's likely what many will search for to begin with.